### ISSUE TITLE
Confusion about the naming and semantics of h_pos vs h_pretext in patch embedding selection

### ISSUE THREAD

**Issue by oldFLAME20** [jinnnju/PaAno#3]
Hi ，thank you for your paper and the code.
I have a question about the semantics of the three embedding slices in the training code:

h_anchors = all_embeddings[:M]
h_pos     = all_embeddings[M:2*M]
h_pretext = all_embeddings[2*M:3*M]

h_pos is constructed from patches that are temporally adjacent to the anchors (±2 steps), which means they are actually consecutive/neighboring in time.
h_pretext is constructed from patches that are 64 steps before the anchors (pretext_step=64), which means they are not adjacent to the anchors in time.
Confusion:
h_pos (positive sample for triplet loss) is the truly consecutive/neighboring patch in time.
h_pretext (positive sample for pretext task) is the non-adjacent patch (separated by 64 steps).
Could it be possible that the code is inconsistent with the paper description, or that the two samples are accidentally reversed in the implementation?
I would really appreciate it if you could clarify this. Thanks again for your great work!
